charts/airbyte/Makefile

```
@@ -0,0 +1,34 @@
+RELEASE_NAME := airbyte
+NAMESPACE := airbyte-test
+
+lint:
+	helm lint .
+
+
+install:
+	helm upgrade --install \
+				--create-namespace \
+				--namespace $(NAMESPACE) \
+				--debug \
+				--wait \
+				--timeout 300s \
```

```suggestion
				--timeout 600s \
```

We've seen some long download times for some users doing something locally.